### PR TITLE
New Serverless Pattern - cdk s3 lambda sns python

### PR DESCRIPTION
**Pull request by pdineshbalaaji**
*Issue #, if available:*

*Description of changes:*

This pattern creates a Amazon S3 bucket, a Lambda function, and an Amazon SNS topic. The Lambda function is triggered by S3 'OBJECT_CREATED' events. The Lambda function can optionally apply business logic to the S3 object. After processing, the Lambda function publishes an SNS notification of the new object.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by cornelcroi** [comment]
Hi,
Thank you for submitting the pattern.
The title in `example-pattern.json` must have maximum length of 75 characters.
Can you reword it to be shorter ?

**Comment by pdineshbalaaji** [comment]
@cornelcroi - made the required changes

**Review by cornelcroi — APPROVED** [review]
